src/config.inc

```diff
@@ -76,7 +76,7 @@ endif
 OSX_IDENTITY="Developer ID Application: Daniel Kroening"
 
 # Detailed version information
-CBMC_VERSION = 5.12
+CBMC_VERSION = 5.12.6
```

Comment:
Shouldn't the git tag be the single source of truth?

Reply:
@peterschrammel We currently use both this and the git tag. We could alternatively change how we do versioning to only take the git tag into consideration; The only problem is then what version do we put on code built e.g. from a source tarball (though I suppose that could be fixed by making sure `version.cpp` is included with that?).

Reply:
I think we need to keep the version number in `src/config.inc`, as long as we want the ability to build cbmc from a downloaded tar ball. Automatically fetching the version number from git as the single source of truth would mean that we can only build from a git checkout.